### ISSUE TITLE
CData support and self closed nodes?

### ISSUE THREAD

**Issue by smith288** [FabianLauer/vs-code-xml-format#2]
Formatting seems to rip out anything after and including a <![CDATA[String goes here]]> as well as a node that is self closed <NodeHere />

**Comment by Morgy93** [comment]
Comments are also cut out. For example:
Before:

```
<reference name="test">
    <!-- <block name="i_wont_stay" /> -->
</reference>
```

After:

```
<reference name="test" />
```

**Comment by FabianLauer** [comment]
Thanks for reporting 👍. I'll try to get the CDATA problem fixed by the end of the week (doctypes are also cut out).
@Morgy93 As far as comments are concerned, IDK whether the XML parser actually supports this (https://github.com/segmentio/xml-parser), so I'll see what I can do and keep you up to date.

**Comment by FabianLauer** [comment]
This should be fixed with version 0.1.1. Just update the extension via VS Code and you'll have it.
As I wrote a while ago, the parser I used for previous versions of the extension didn't support comments and cdata nodes. I started work on [tsxml](https://github.com/FabianLauer/tsxml) a few weeks ago though, which supports all XML node types and is the XML library behind this extension since version 0.1.1.
Sorry to keep you waiting :wink:
